config/jobs/kubernetes/sig-cluster-lifecycle/manifests.yaml

```diff
@@ -13,8 +13,7 @@ periodics:
       - --scenario=execute
       - --
       - ./tests/e2e/manifests/verify_manifest_lists.sh
-      image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
-      imagePullPolicy: Always
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20201201-1941b8f-master
```

Comment:
The bot seems to update the image tag automatically #20115 
Now the latest image is `gcr.io/k8s-testimages/kubekins-e2e:v20201203-4778e22-master`

Reply:
good catch @SataQiu 
updated to `v20201203-4778e22-master`